Dockerfile

```
@@ -1,7 +1,9 @@
 FROM ruby:2.7.0
 
 RUN apt-get update && apt-get install -qq -y --no-install-recommends \
-      build-essential nodejs libpq-dev
+      build-essential nodejs libpq-dev cron && rm -rf /var/lib/apt/lists/*
```

```suggestion
     build-essential nodejs libpq-dev cron && rm -rf /var/lib/apt/lists/*
```
Ficou desalinhado do RUN